test/scripts/deviceshifu-demo-aio.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+# 2022 Edgenesis Inc.
+set -e
+
+if [ $# -eq 0 ]; then
+    echo "No arguments provided, use 'build_demo' or 'run_demo'"
+    exit 1
+fi
+
+SHIFU_IMG_VERSION=v0.2.1
```

Comment:
is it possible to get the tag from `version.txt` ?

Reply:
It seems can't, because installing shifu demo using a script won't have version.txt

Reply:
I see, in this case this should be `nightly` for `main` branch.
We need to update `make tag` to update this or take this as an argument to build for the correct Shifu release